### PR TITLE
Update ShEx for Disease History and Diagnosis

### PR DESCRIPTION
**Pull request by sxzhang1201**
Main changes including:
- remove prov:informedBy triple 
- add "so:has-input" triple
- make choices within the "process"

### CODE REVIEW

File: shex/sio-model/diseaseHistoryAndDiagnosisShape.shex [hunk 1]
```diff
@@ -19,19 +18,32 @@ PREFIX loinc: <http://purl.bioontology.org/ontology/LNC/>
...
+  a [sio:SIO_001001];
+  sio:SIO_000229 @:outputReferToDiseaseShape
+  );
+  sio:SIO_000680 xsd:dateTime
 }
 
 :outputShape IRI {
   a [sio:SIO_000340];
-  (a [loinc:LP91394-4]|
+  a [loinc:LP91394-4]
```
Comment (rajaram5) [inline shex/sio-model/diseaseHistoryAndDiagnosisShape.shex:33]: This should be list of terms. See the previous version https://github.com/ejp-rd-vp/CDE-semantic-model/blob/develop/shex/diseaseHistoryAndDiagnosisShape.shex#L36

**Review by rajaram5 — CHANGES_REQUESTED** [review]
Missing types in output node

**Comment by pabloalarconm** [comment]
Apart from Rajaram's comment, no more changes from my side, thank you Shuxin

File: shex/sio-model/diseaseHistoryAndDiagnosisShape.shex (hunk 1 above)
Comment (sxzhang1201) [reply shex/sio-model/diseaseHistoryAndDiagnosisShape.shex]: Thanks for identifying that. Finish updating the type.

**Review by rajaram5 — APPROVED** [review]
Looks good to me

**Pull request merged**